src/odemis/driver/orsay.py

```diff
@@ -2605,7 +2612,7 @@ def _changePower(self, goal):
         :param (float) return: goal
         """
         logging.debug("Turning Chamber light %s." % "on" if goal[0] else "off")
-        self._parameter.Target = goal[0] != 0.0
+        self._parameter.Target = int(goal[0])
```

Comment:
The parameter is apparently a boolean, so it'd make more sense to pass a boolean. Also, `_changePower()` is a setter, so it should return the actual value set. So I'd suggest this code:
```python
power = bool(goal[0])
logging.debug("Turning Chamber light %s." % "on" if power else "off")
self._parameter.Target = power
return [1.0 if power else 0.0]
```

Reply:
It actually is a string of an integer. But setting it with an int also works. Only the values 0 or 1 are allowed. The values True/False actually don't work. So you suggestion doesn't work.